examples/czmq/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <czmq.h>
+
+
+int main()
+{
+	zsock_t *push = zsock_new_push ("inproc://example");
```

Comment:
@ErniBrown it's an example so it's not so critical, however try to avoid tabs, see:
* http://0.readthedocs.io/en/latest/git.html#never-use-tabs